docs/search/criteria_reference/order_search_criteria.md

```diff
@@ -5,7 +5,7 @@ edition: commerce
 
 # Order Search Criteria reference
 
-Order Search Criteria are part of [Search Criteria](search_criteria_reference.md) and they are supported only by Order Search.
+Order Search Criteria are supported only by [Order Search (`OrderService::findOrders`)](order_management_api.md#get-multiple-orders).
```

Comment:
```suggestion
Order Search Criteria are only supported by [Order Search (`OrderService::findOrders`)](order_management_api.md#get-multiple-orders).
```
Same comment for all the below changes

Reply:
Done in 7afed9e5a740bc3bcadbe279edbd9f4b28ec05c7